src/server/auth/cmds/configure.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/pachyderm/pachyderm/src/client/auth"
 	"github.com/pachyderm/pachyderm/src/client/pkg/grpcutil"
 	"github.com/pachyderm/pachyderm/src/server/pkg/cmdutil"
+	"gopkg.in/pachyderm/yaml.v3"
```

Comment:
Is there a reason this needs to be imported as `gopkg.in/pachyderm/yaml.v3` rather than as `github.com/pachyderm/yaml`?

Reply:
Basically, we have to use `gopkg.in` because `github.com/go-yaml/yaml` used `gopkg.in`.

In detail, it has to do with the branch structure. `v3` is a branch in `pachyderm/yaml` (as it is in `go-yaml/yaml`, because we forked it). When you import this as `gopkg.in/pachyderm/yaml.v3`, the `gopkg.in` service converts the `v3` suffix into a branch, and causes our repo to depend on the right branch. http://labix.org/gopkg.in has more info on how `gopkg.in` works, but it's basically a proxy that does this conversion.